### PR TITLE
Add Starship

### LINKED ISSUE #4488: Starship
Name: Starship
Website: starship.rs
Alexa rank: #769,216
Official resources for icon and color:  #b30bb0 (https://github.com/starship/starship)

Starship is a Cross-shell command prompt. Is open-source and has a decent amount contributors on github repository. Also has 13.2k stars.

### PR DESCRIPTION
**Pull request by klu17**
**Issue:** Closes #4488
**Name:** Starship
**Website:** starship.rs
**Alexa rank:** 769,216

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  -  **Why did you pick the hex value?:** @skywarth mentioned `#B30BB0` and `#DD0B78` was used on the website. It appears that both colors are used in the gradient. I went with the last one mentioned in the issue: `#DD0B78`.
  - I manually combined the paths with Inkscape. Then, I ran it through SVGOMG

### Preview
image

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @klu17! 🎉
Just taking a quick look at it, it appears there are some slight differences between your SVG (red) and source (black). They could be as a result of the optimization process, but could you take another look - and optimize to 4 decimals if need be?

image

File: icons/starship.svg
Comment (adamrusted) [inline icons/starship.svg:1]: This version is still showing some differences... :thinking:

I'll take a look and see if I'm able to get a path made up for this...

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
Could another @simple-icons/maintainers take a look at this? The path I've included should be fine - but would be good to get your input before merging.

**Comment by mondeja** [comment]
I'm able to optimize with 3 decimals and without seeing differences against the original resized to 24px. This is the path I got:

```svg
M12 0A12 12 0 0 0 0 12a12 12 0 0 0 4.684 9.512c4.1-.44 5.598-3.67 5.598-3.67l3.226 1.053s-.53 3.17-2.547 5.06A12 12 0 0 0 12 24a12 12 0 0 0 12-12A12 12 0 0 0 12 0zM6.597 4.575c.146 1.2.8 1.9 2.03 2.03-1.364.208-2.05.88-2.03 2.033-.2-1.114-.863-1.805-2.03-2.032 1.26-.15 1.89-.872 2.03-2.03zm8.584.992c1.626 1.994 2.345 3.763 1.234 6.246.92.997 1.314 2.598 1.3 4.094l-2.825-.65-.444 1.18-3.637-1.062.367-1.267-2.78-1.103c.887-1.083 1.773-1.615 3.148-2.29-.123-2.125 1.593-3.648 3.636-5.148zm-.716 2.995c-.584 0-1.057.474-1.057 1.058 0 .584.473 1.057 1.057 1.057s1.057-.473 1.057-1.057-.472-1.058-1.056-1.058z
```

Could you check it @adamrusted?

**Comment by adamrusted** [comment]
> I'm able to optimize with 3 decimals and without seeing differences against the original resized to 24px. This is the path I got:
> 
> ```svg
> M12 0A12 12 0 0 0 0 12a12 12 0 0 0 4.684 9.512c4.1-.44 5.598-3.67 5.598-3.67l3.226 1.053s-.53 3.17-2.547 5.06A12 12 0 0 0 12 24a12 12 0 0 0 12-12A12 12 0 0 0 12 0zM6.597 4.575c.146 1.2.8 1.9 2.03 2.03-1.364.208-2.05.88-2.03 2.033-.2-1.114-.863-1.805-2.03-2.032 1.26-.15 1.89-.872 2.03-2.03zm8.584.992c1.626 1.994 2.345 3.763 1.234 6.246.92.997 1.314 2.598 1.3 4.094l-2.825-.65-.444 1.18-3.637-1.062.367-1.267-2.78-1.103c.887-1.083 1.773-1.615 3.148-2.29-.123-2.125 1.593-3.648 3.636-5.148zm-.716 2.995c-.584 0-1.057.474-1.057 1.058 0 .584.473 1.057 1.057 1.057s1.057-.473 1.057-1.057-.472-1.058-1.056-1.058z
> ```
> 
> Could you check it @adamrusted?

I've pushed a commit with a 3 decimal path above, could you check it and merge if it's good @mondeja?

**Review by mondeja — APPROVED** [review]
>I've pushed a commit with a 3 decimal path above, could you check it and merge if it's good @mondeja?

Oh, I didn't see the complete commit message :man_facepalming: Yes, It's perfect!
Thank you @adamrusted and @klu17 for this contribution! :pray:

**Pull request merged**
